java/src/test/java/org/rocksdb/RocksDBTest.java

```diff
@@ -1185,7 +1184,7 @@ public void getApproximateSizes() throws RocksDBException {
 
         final long[] sizes = db.getApproximateSizes(
             Arrays.asList(
-                new Range(new Slice(key1), new Slice(key2)),
+                new Range(new Slice(key1), new Slice(key1)),
```

Comment:
Why the change in the range, is it a genuine mistake originally?

Reply:
Because test is asserting against 0 size. It cannot happen it is zero size when it gets range that is non empty.

Reply:
Okie dokie :-)